cvrf2csaf.py

```diff
@@ -106,8 +106,8 @@ def create_json(self):
 
 # Load CLI args
 parser = argparse.ArgumentParser(description='Converts CVRF XML input into CSAF 2.0 JSON output.')
-parser.add_argument('--input_file', dest='input_file', type=str, help="CVRF XML input file to parse", default='./sample_input/sample.xml')
-parser.add_argument('--out_file', dest='out_file', type=str, help="CVRF JSON output file to write to.", default='./output/sample.json')
+parser.add_argument('--input-file', dest='input_file', type=str, help="CVRF XML input file to parse", default='./sample_input/sample.xml')
```

Comment:
Another user experience improvement might be to add `metavar='PATH'` for in and out path parameter declarations.